certbot/certbot/_internal/client.py

```diff
@@ -32,13 +33,23 @@
 
 logger = logging.getLogger(__name__)
 
-
 def acme_from_config_key(config, key, regr=None):
     "Wrangle ACME client construction"
     # TODO: Allow for other alg types besides RS256
     net = acme_client.ClientNetwork(key, account=regr, verify_ssl=(not config.no_verify_ssl),
-                                    user_agent=determine_user_agent(config))
-    return acme_client.BackwardsCompatibleClientV2(net, key, config.server)
+                                user_agent=determine_user_agent(config))
```

Comment:
*extremely* minor nit, but to keep the git blame accurate, want to put the spaces back?